.travis.yml

```diff
@@ -6,10 +6,8 @@ cache:
 env:
   - GO111MODULE=on
 
-notifications:
```

Comment:
This was a Slack instance that was just for Pion, hasn't been used for almost two years :)

We moved everything to Gophers Slack since it had paid features.